autoendpoint/src/extractors/user.rs

```diff
@@ -69,7 +69,7 @@ async fn validate_webpush_user(
 }
 
 /// Drop a user and increment associated metric
-async fn drop_user(uaid: Uuid, ddb: &dyn DbClient, metrics: &StatsdClient) -> ApiResult<()> {
+pub async fn drop_user(uaid: Uuid, ddb: &dyn DbClient, metrics: &StatsdClient) -> ApiResult<()> {
```

Comment:
Out of curiosity, why is `pub` added here?

Reply:
It was needed on an earlier version of this PR, the `pub` can be removed now (though it's not terribly important)